bosh-director/lib/bosh/director/api/stemcell_manager.rb

```diff
@@ -31,16 +31,22 @@ def stemcell_exists?(name, version)
         false
       end
 
-      def create_stemcell_from_url(username, stemcell_url)
-        JobQueue.new.enqueue(username, Jobs::UpdateStemcell, 'create stemcell', [stemcell_url, { remote: true }])
+      def create_stemcell_from_url(username, stemcell_url, stemcell_sha=nil)
```

Comment:
i dont think =nil is necessary since we always call the method with 3 params. also may have to adjust tests to call with 3 params.